modules/openapi-generator/src/main/resources/typescript-fetch/apis.mustache

```diff
@@ -193,9 +193,7 @@ export class {{classname}} extends runtime.BaseAPI {
         if (this.configuration && this.configuration.accessToken) {
             // oauth required
             if (typeof this.configuration.accessToken === 'function') {
-                headerParameters["Authorization"] = this.configuration.accessToken("{{name}}", [{{#scopes}}"{{{scope}}}"{{^-last}}, {{/-last}}{{/scopes}}]);
-            } else {
```

Comment:
looks like the `else` condition got lost - was it intentional?

Reply:
Intentional. Code flow analysis denoted type of the **accessToken** in else to *never*. **BaseAPI** getter returns a function in any case, so logically *else* should never happen.

Reply:
do we still need the `if` condition then? if we're convinced it's always true

Reply:
Good point, no we don't since the first if should take care of undefined case. I will fix